manifests/php.yml

```diff
@@ -72,7 +72,6 @@ tests/:
     test_dynamic_configuration.py:
       TestDynamicConfigV1: missing_feature
       TestDynamicConfigV2: missing_feature
-    test_otel_span_methods.py: missing_feature
```

Comment:
Could you rather set the version where the feature is supposed to be ok ?

Reply:
Set the version for the three Otel test files to v0.94.0 or later, with the exception of three tests (the infamous `analytics.event` override) to v0.96.0+ (last release) 😃